src/components/TeamPlayer.js

```diff
@@ -0,0 +1,255 @@
+        id: team.id = 1,
+        city: team.city = "Oklahoma",
+        teamName: team.teamName = "Thunder",
+        conf: team.conf = "west",
+        logo: team.logo = require('../assets/i.png'),
+        player: [
+            {
+                id: player.id = 1,
+                name: player.name = "Russel Westbrook",
+                salary: player.salary = "$23M/Year",
+                careerPoints: player.careerPoints = "23,000 Points"
+            },
+            {
+                id: player.id = 2,
+                name: player.name = "Paul George",
+                salary: player.salary = "$21M/Year",
+                careerPoints: player.careerPoints = "20,000 Points"
+            },
+            {
+                id: player.id = 3,
+                name: player.name = "Steve Adams",
+                salary: player.salary = "$13M/Year",
+                careerPoints: player.careerPoints = "8,000 Points"
+            }
+        ]
+    },
+    {
+        id: team.id = 2,
+        city: team.city = "Chicago",
+        teamName: team.teamName = "Bulls",
+        conf: team.conf = "East",
+        logo: team.logo = require('../assets/bulls.jpg'),
+        player: [
+            {
+                id: player.id = 1,
+                name: player.name = "Michael Jordan",
+                salary: player.salary = "$23M/Year",
+                careerPoints: player.careerPoints = "34,000 Points"
+            },
+            {
+                id: player.id = 2,
+                name: player.name = "Pippen Scott",
+                salary: player.salary = "$21M/Year",
+                careerPoints: player.careerPoints = "20,000 Points"
+            },
+            {
+                id: player.id = 3,
+                name: player.name = "Steve Kerr",
+                salary: player.salary = "$13M/Year",
+                careerPoints: player.careerPoints = "8,000 Points"
+            }
+        ]
+    },
+    {
+        id: team.id = 3,
+        city: team.city = "Golden State",
+        teamName: team.teamName = "Warriors",
+        conf: team.conf = "West",
+        logo: team.logo = require('../assets/GSW.png'),
+        player: [
+            {
+                id: player.id = 1,
+                name: player.name = "Stephen Curry",
+                salary: player.salary = "$47M/Year",
+                careerPoints: player.careerPoints = "34,000 Points"
+            },
+            {
+                id: player.id = 2,
+                name: player.name = "Draymond Green",
+                salary: player.salary = "$21M/Year",
+                careerPoints: player.careerPoints = "20,000 Points"
+            },
+            {
+                id: player.id = 3,
+                name: player.name = "klay Thompson",
+                salary: player.salary = "$13M/Year",
+                careerPoints: player.careerPoints = "8,000 Points"
+            }
+        ]
+    },
+    {
+        id: team.id = 4,
+        city: team.city = "Los angeles",
```

Comment:
do you need to use `team` inside this object?

Reply:
it's because i have a team class but i can assign values into json without using objects

Reply:
Avoid doing that, you are overriding the value of the team class so it's pretty much useless to add it there

Reply:
so in that case the team.json file will be a simple json without including other objects?